pkg/apis/keycloak/v1alpha1/keycloakclient_types.go

```diff
@@ -164,6 +164,9 @@ type KeycloakAPIClient struct {
 	// Authorization settings for this resource server.
 	// +optional
 	AuthorizationSettings *KeycloakResourceServer `json:"authorizationSettings,omitempty"`
+	// Service account roles for this client.
+	// +optional
+	ServiceAccountRoles []string `json:"serviceAccountRoles,omitempty"`
```

Comment:
Is this Realm Roles or Client Roles? I think we should clearly differentiate this, similarly to [what we have for users](https://github.com/keycloak/keycloak-operator/blob/537d3ea9fb71570a4b7eb51fb6460a28b309450c/pkg/apis/keycloak/v1alpha1/keycloakuser_types.go#L71-L76) (because service account _is_ a user).